BUILD.bazel

```diff
@@ -105,6 +105,23 @@ cc_library(
     ],
 )
 
+cc_library(
+    name = "core_worker_lib",
```

Comment:
how about just calling this worker_lib and the directory `src/ray/worker/`?

Reply:
The main reason why I call it `core_worker` is because we may have a `cpp_worker` directory in the future, and I want to distinguish them clearly.

Reply:
`cmmon_worker` looks fine as well. I don't have strong preference. how do others think?

Reply:
I also don't have a strong preference, `common_worker` is fine with me.

Reply:
Discussed offline. Both options sound fine. I'll keep `core_worker`.